x-pack/plugins/security/server/plugin.test.ts

```diff
@@ -71,14 +71,10 @@ describe('Security Plugin', () => {
                 "authc": Object {
                   "createAPIKey": [Function],
                   "getCurrentUser": [Function],
-                  "getSessionInfo": [Function],
```

Comment:
__note:__ technically it's a breaking change (if we already started to treat changed plugin contracts as such), but nobody in Kibana was using it and I'm not sure we want to expose this to 3rd party plugins yet. Starting from 8.0 I'd not do that :slightly_smiling_face: